components/aws/sagemaker/test/integration/.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
```

Comment:
Just to explain, our unit tests are currently under `/tests/unit_tests/`. In keeping with that format, `/tests/integration_tests/` would be the most logical. It is also necessary to keep them similar so that our CodeBuild build specs know where to look.

Reply:
will do